### ISSUE TITLE
Use standard java logging mechanism

### ISSUE THREAD

**Issue by nicolaichuk** [dragome/dragome-sdk#170]
@fpetrola, Why you write own com.dragome.compiler.utils.Log instead of using standard java.util.logging.Logger?

**Comment by nicolaichuk** [comment]
relate https://github.com/dragome/dragome-sdk/issues/169

**Comment by fpetrola** [comment]
This is inherited from j2js project

**Comment by nicolaichuk** [comment]
It will be good replace it to java.util.logging.Logger for more control over log process.

**Comment by fpetrola** [comment]
I'd rater use SL4J which is an abstract layer for logging, it's not dependent of any particular logging framework.

**Comment by nicolaichuk** [comment]
I thing you mean [slf4j](https://www.slf4j.org/). It is possible and SL4J. It does not matter to me.

**Comment by fpetrola** [comment]
ok, if we use slf4j we should find a slf4j backend to run also at client side, may be it's easier as log in console triggered by a system.out.println

**Comment by fpetrola** [comment]
May be this one: https://www.slf4j.org/api/org/slf4j/impl/SimpleLogger.html

**Comment by nicolaichuk** [comment]
Yes. I think capabilities of SimpleLogger will be enough for now.
